2022/21xxx/CVE-2022-21504.json

```diff
@@ -1,18 +1,75 @@
+
```

Comment:
('Expecting property name enclosed in double quotes: line 72 column 13 (char 3110)',)